client-dotnet/src/EvoMaster.Controller.Api/dto/UnitsInfoDto.cs

```diff
@@ -11,7 +11,7 @@ public class UnitsInfoDto {
         /**
      * Then name of all the units (eg classes) in the SUT
      */
-        public ISet<string> UnitNames = new HashSet<string>();
+        public IEnumerable<string> UnitNames { get; set; }
```

Comment:
UnitsInfoRecorder already stores the unit names inside a ConcurrentHashSet which applies the uniqueness

Reply:
The main reason for changing this was that `_unitNames `inside `UnitsInfoRecorder.cs` is of type `ConcurrentHashSet` which is defined in `Client.Util`. Those class doesn't implement ISet for the sake of simplicity. Maybe a class which functions like a set should implement ISet as HashSet class does but since we didn't need those methods and we only expected a limited set of functionalities from it, I thought that it may be enough for it to only implement generic ICollection and IReadOnlyCollection. 
Besides, the `UnitNames `property has unique values as before.